admin/sql/create_tables.sql

```diff
@@ -1,11 +1,12 @@
 BEGIN;
 
 CREATE TABLE lowlevel (
-  id          SERIAL,
-  mbid        UUID NOT NULL,
-  build_sha1  TEXT NOT NULL,
-  lossless    BOOLEAN                  DEFAULT 'n',
-  submitted   TIMESTAMP WITH TIME ZONE DEFAULT NOW()
+  id              SERIAL,
+  gid             UUID    NOT NULL,
+  build_sha1      TEXT    NOT NULL,
+  lossless        BOOLEAN                  DEFAULT 'n',
+  submitted       TIMESTAMP WITH TIME ZONE DEFAULT NOW(),
+  is_mbid         id_type NOT NULL
```

Comment:
Yes, I see 👍 
The ENUM type is called id_type, should we call it also gid_type?